test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -10007,5 +10007,24 @@ TEST_F(AtenXlaTensorTest, TestEmbeddingBackward) {
   }
 }
 
+TEST_F(AtenXlaTensorTest, TestEarlySyncLiveTensors) {
```

Comment:
It is better to name it TestLocalScalarDense, this way people reading it will know which op to look for.